stylua.toml

```diff
@@ -0,0 +1,5 @@
+indent_style = "Space"
+indent_width = 4
```

Comment:
lets keep this at 2 since that was what it was before and the total diff of this MR will be smaller